mm2src/coins/utxo/swap_proto_v2_scripts.rs

```diff
@@ -1,4 +1,4 @@
-/// This module contains functions building Bitcoins scripts for the "Swap protocol upgrade" feature
+/// This module contains functions building Bitcoins scripts for the "Trading protocol upgrade" feature
```

Comment:
"Trading protocol upgrade" can be misleading depending on the current time.

```suggestion
/// This module contains functions building Bitcoins scripts for trading protocol v2.
```

Reply:
This is the actual feature name https://github.com/KomodoPlatform/komodo-defi-framework/issues/1895. Maybe it's even worth to change filename and other naming for overall consistency :slightly_smiling_face:

Reply:
Good point, I updated issue title and will use v2 wheheever possible.